ui/src/App.tsx

```diff
@@ -57,10 +86,11 @@ function App() {
           </BrowserRouter>
         ) : (
           <>
-            <Message
-              message="The API is not available. Please, check the URL and try again."
-              severity="error"
-            />
+            <Alert severity="error" sx={{ marginTop: 2 }} variant="filled">
```

Comment:
Would be nice to know why exactly: is the network or the web3signer api issue?

Also, the network should be optional and take the default value of "mainnet"

Reply:
Done. https://github.com/dappnode/eth2-keymanager-frontend/commit/b5e1408978d0ebd3e64d1207c11996f5daf04bd1